jenkins_rock/rockcraft.yaml

```diff
@@ -0,0 +1,42 @@
+name: jenkins
```

Comment:
Since there is a rock, could you add */*.rock to the .gitignore?

Reply:
Ops, forgot to format ```*/*.rock```

Reply:
Added, thank you :)